journalpump/senders/aws_cloudwatch.py

```diff
@@ -38,13 +38,18 @@ def _init_logs(self):
         except botocore.exceptions.ClientError as err:
             if err.response["Error"]["Code"] != "ResourceAlreadyExistsException":
                 raise
```

Comment:
The entire init function should probably be wrapped in some error handler that just marks the sender as disconnected in case of any error as these too would cause entire journalpump to fail when there's some problem with CloudWatch config.